lib/pure/asyncdispatch.nim

```diff
@@ -1680,4 +1680,4 @@ proc waitFor*[T](fut: Future[T]): T =
   while not fut.finished:
     poll()
 
-  fut.read
+  fut.read
```

Comment:
@cheatfate what text editor are you using? Whatever it is, it's not adding a newline at the end of files which is incorrect.